src/workerd/api/http.c++

```diff
@@ -1176,6 +1178,41 @@ void Request::shallowCopyHeadersTo(kj::HttpHeaders& out) {
 }
 
 kj::Maybe<kj::String> Request::serializeCfBlobJson(jsg::Lock& js) {
+  if (cacheMode != CacheMode::NONE &&
+      util::Autogate::isEnabled(util::AutogateKey::HTTP_REQUEST_CACHE)) {
```

Comment:
Given that setting of `cacheMode` is already behind an experimental compatibility flag, it seems unnecessary to also use an autogate here.